libs/designer-ui/src/lib/mixedinputeditor.tsx

```diff
@@ -0,0 +1,110 @@
+import type React from 'react';
+import { useEffect, useState } from 'react';
+import { Toggle } from '@fluentui/react/lib/Toggle';
+import type { ChangeHandler, ValueSegment } from '@microsoft/logic-apps-shared';
+import { SchemaEditor } from './schemaeditor';
+import { FloatingActionMenuInputs } from './floatingactionmenu/floatingactionmenuinputs';
+
+interface MixedInputEditorProps {
+  initialValue: ValueSegment[];
+  onChange: ChangeHandler;
+  supportedTypes?: string[];
+  isRequestApiConnectionTrigger?: boolean;
+  label?: string;
+  readonly?: boolean;
+  useStaticInputs: boolean;
+}
+
+// Function to check if `initialValue` contains "useSchemaEditor": true
+const checkIsSchemaEditor = (initialValue: ValueSegment[]): boolean => {
+  return initialValue.some((segment) => {
+    try {
+      const parsedValue = JSON.parse(segment.value);
+      return parsedValue?.useSchemaEditor === true;
+    } catch (error) {
+      console.error('Error parsing value segment:', error);
+      return false;
+    }
+  });
+};
+
+// Function to add "useSchemaEditor": true if it's missing
+const addUseSchemaEditor = (value: ValueSegment[]): ValueSegment[] => {
```

Comment:
Same thing here, if you write some unit tests for this it would be easier to test in isolation if the function exists in a file where it can be exported